Makefile

```diff
@@ -279,6 +279,7 @@ clean:
 	rm -rf arbitrator/wasm-testsuite/tests
 	rm -rf $(output_root)
 	rm -f contracts/test/prover/proofs/*.json contracts/test/prover/spec-proofs/*.json
+	rm -f contracts-legacy/test/prover/proofs/*.json contracts-legacy/test/prover/spec-proofs/*.json
```

Comment:
I presume something in the Arbitrator tests is not reading the correct json ABI for initialization